packages/code-du-travail-modeles/src/modeles/common/seniority.ts

```diff
@@ -72,9 +74,14 @@ export abstract class SeniorityDefault<T extends SupportedCc>
   computeRequiredSeniority({
     dateEntree,
     dateNotification,
+    dateArret,
     absencePeriods = [],
   }: DefaultSeniorityRequiredProps): RequiredSeniorityResult {
-    return this.compute(dateEntree, dateNotification, absencePeriods);
+    return this.compute(
+      dateEntree,
+      dateArret || dateNotification,
```

Comment:
```suggestion
      dateArret ?? dateNotification,
```

d'un point de vue métier, c'est validé ? Faut-il attendre le retour de Virginie ou alors celui de Berengere s'il a discuté avec le métier

ça serait intéressant de rajouter un test côté modeles pour garder cette règle là

Reply:
En fait, j'avais mis ?? au départ mais l'ia m'a indiqué que je ne gère pas le cas de string vide. L'avantage du || c'est qu'il gère plus de cas que le ?? (undefined, string vide et null)